harikath-chicago.md

```diff
@@ -0,0 +1,5 @@
+# Chicago
+
+### Places to eat
```

Comment:
@Harikath I would :heart: some recommendations for great Indian food in the area! I am craving a Dosa :wink: